pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"k8s.io/kubernetes/pkg/scheduler/util"
 )
 
+var negPriority, lowPriority, midPriority, highPriority, veryHighPriority = int32(-100), int32(0), int32(100), int32(1000), int32(10000)
```

Comment:
where is this line coming from ?

Reply:
It's from core/generic_scheduler_test.go L798.

Right now the constants are private, I don't think it's worth making them public and introduce a new dependency to internal package.

Reply:
"A little copying is better than a little dependency."
https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s